vms/proposervm/height_indexed_vm.go

```diff
@@ -85,32 +33,7 @@ func (vm *VM) GetBlockIDAtHeight(ctx context.Context, height uint64) (ids.ID, er
 	}
 }
 
-// As postFork blocks/options are accepted, height index is updated even if its
-// repairing is ongoing. vm.ctx.Lock should be held
```

Comment:
removed comment since it was only explaining the extra complexity from potential ongoing repair/checkpointing